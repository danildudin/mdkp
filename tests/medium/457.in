28 8 0
21 506 752 614 110 976 669 886 505 978 921 286 565 249 875 75 150 99 101 745 625 31 105 291 439 237 334 806 
849 857 121 290 482 671 116 400 820 208 938 944 682 373 779 487 614 725 611 135 657 220 929 483 994 77 635 474 
99 930 2 449 575 184 981 653 818 328 267 686 165 329 484 995 379 383 807 884 462 230 222 962 152 592 292 103 
724 90 799 143 853 798 661 831 988 170 651 701 53 810 201 319 946 487 330 880 92 292 349 663 649 368 75 982 
58 176 405 756 596 249 579 889 7 681 824 849 85 665 59 751 762 425 7 19 461 486 326 953 862 402 41 173 
662 624 675 945 761 449 486 792 554 828 299 377 648 292 460 224 216 253 929 962 498 488 578 852 114 195 921 33 
654 239 459 200 459 123 935 314 167 636 35 953 936 739 282 813 376 773 449 966 504 49 277 45 19 327 180 683 
835 582 697 942 705 22 977 114 267 402 171 594 819 74 195 826 860 433 331 268 532 164 297 36 641 808 147 31 
131 388 332 824 691 250 913 393 113 188 389 670 276 175 636 765 910 634 799 765 67 12 851 569 260 906 737 225 
5399 7206 9640 6196 7479 5287 6217 9311 
