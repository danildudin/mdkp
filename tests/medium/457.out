9138
12
2 3 5 6 7 9 10 14 19 20 23 27 
